src/transformers/generation_utils.py

```diff
@@ -359,12 +359,69 @@ class BeamSampleEncoderDecoderOutput(ModelOutput):
 BeamSampleOutput = Union[BeamSampleEncoderDecoderOutput, BeamSampleDecoderOnlyOutput]
 
 
+ENCODER_MODEL_INPUT_NAMES = ["input_ids", "inputs_embeds", "input_values", "input_features", "pixel_values"]
```

Comment:
This are all possible encoder input names I think

Reply:
There aren't more than those for vision models no? @NielsRogge

Reply:
Indeed!